octoprint_mrbeam/iobeam/dust_manager.py

```diff
@@ -254,12 +271,19 @@ def _finish_test_fan_rpm(self):
                 else:
                     rpm_average = -1
 
+                if len(self._last_pressure_values):
+                    pressure_average = sum(self._last_pressure_values) /len(self._last_pressure_values)
```

Comment:
I'm curious , why we need to calculate the average and then log to analytics. wont we benefit more of the data if we log all of it and then on the cloud part we have `Raw Data` that we can see and analyse the way we see fit ?

Reply:
It will be more data, but you are right, we can analyse it better with more data